src/plugins/console/server/lib/spec_definitions/json/overrides/indices.simulate_template.json

```diff
@@ -0,0 +1,16 @@
+{
+  "indices.simulate_template": {
```

Comment:
This file isn't needed, since autocomplete of both `POST _index_template/_simulate` and `POST _index_template/_simulate/{template}` will be handled by `indices.simulate_index_template.json`.

Reply:
Thanks. Done.